module exceptions

import error
import io

try {
  None.non_existent_field ; Will trigger an exception.
} catch e {
  io.fprintln(io.ERROR, e)
}

try {
  raise error.Error('Hello, world!')
} catch e {
  io.fprintln(io.ERROR, e)
}